charts/astronomer/values.yaml

```diff
@@ -311,6 +311,19 @@ houston:
     #   - 2.0.0-alpine3.10-onbuild
     #   channel: stable
 
+  RuntimeReleasesConfig:
+    # example of usage:
+    # available_releases:
+    #runtimeVersions:
+    #  4.2.6:
+    #    metadata:
+    #      airflowVersion: 2.2.5
+    #      channel: stable
+    #      releaseDate: '2022-04-19'
+    #    migrations:
+    #      airflowDatabase: false
```

Comment:
What does this field do? Users will want to know when it needs to be included and if it can be omitted.